<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dlg_record_nibp</class>
 <widget class="QDialog" name="Dlg_record_nibp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>570</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>无创血压回顾</string>
  </property>
  <widget class="QPushButton" name="m_btn_return">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>460</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_btn_forwd_end">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>420</y>
     <width>51</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>&gt;|</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_btn_back_end">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>420</y>
     <width>51</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>|&lt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_btn_back">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>420</y>
     <width>51</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_btn_back_fast">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>420</y>
     <width>51</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;&lt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_btn_forwd">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>420</y>
     <width>51</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_btn_forwd_fast">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>420</y>
     <width>51</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>&gt;&gt;</string>
   </property>
  </widget>
  <widget class="QTableView" name="tv_nibp">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>550</width>
     <height>360</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
